console/services/app_config/component_graph.py

```diff
@@ -134,6 +134,17 @@ def bulk_create(self, component_id, graphs):
                 ))
         ComponentGraph.objects.bulk_create(cgs)
 
+    @transaction.atomic
+    def batch_delete(self, component_id, graph_ids):
+        component_graph_repo.batch_delete(component_id, graph_ids)
+        # rearrange
+        graphs = component_graph_repo.list(component_id)
+        sequence = 0
+        for graph in graphs:
```

Comment:
Not very efficient?

Reply:
Fortunately, the data volume of the chart will not be too large. This is also acceptable.

Reply:
Improvement: rearrange only if the serial number is greater than 10000